src/bind.py

```diff
@@ -154,18 +158,68 @@ def _to_bind_zones(self, record_requirer_data: DNSRecordRequirerData) -> typing.
                 zones_entries[entry.domain] = []
             zones_entries[entry.domain].append(entry)
 
-        zones_content: typing.Dict[str, str] = {}
-        for zone, entries in zones_entries.items():
-            content = constants.ZONE_HEADER_TEMPLATE.format(zone=zone, serial=int(time.time()))
+        zones: typing.List[Zone] = []
+        for domain, entries in zones_entries.items():
+            zone = Zone(domain=domain, entries=[])
             for entry in entries:
+                zone.entries.append(create_dns_entry_from_requirer_entry(entry))
+            zones.append(zone)
+        return zones
+
+    def _get_conflicts(
+        self, zones: typing.List[Zone]
+    ) -> typing.Tuple[typing.Set[DnsEntry], typing.Set[DnsEntry]]:
+        """Return conflicting and non-conflicting entries.
+
+        Args:
+            zones: list of the zones to check
+
+        Returns:
+            A tuple containing the non-conflicting and conflicting entries
+        """
+        entries = [e for z in zones for e in z.entries]
+        nonconflicting_entries: typing.Set[DnsEntry] = set()
+        conflicting_entries: typing.Set[DnsEntry] = set()
+        while entries:
+            entry = entries.pop()
+            found_conflict = False
+            if entry in conflicting_entries:
+                continue
+            for e in entries:
+                if entry.conflicts(e) and entry != e:
+                    found_conflict = True
+                    conflicting_entries.add(entry)
+                    conflicting_entries.add(e)
+
+            if not found_conflict:
+                nonconflicting_entries.add(entry)
+
+        return (nonconflicting_entries, conflicting_entries)
+
+    def _zones_to_files(self, zones: typing.List[Zone]) -> typing.Dict[str, str]:
```

Comment:
nitpick (feel free to resolve!)
```suggestion
    def _zones_to_file_contents(self, zones: typing.List[Zone]) -> typing.Dict[str, str]:
```

Reply:
Fixed